<!--xnode_vehicle.launch-->
<launch> 
        <node name="xnode_vehicle" pkg="xpkg_vehicle" type="xnode_vehicle" output="screen" >
            <param name="ini_path" value="$(find xpkg_vehicle)/ini/device_id_list.ini"/>
            <param name="show_loc" value="true"/>
            <param name="calc_speed" value="false"/>
            <param name="mode_lock" value="true"/>
            <param name="show_path" value="true"/>
            <param name="rate_x" value="1.0"/>
            <param name="rate_y" value="1.0"/>
            <param name="rate_z" value="1.0"/>
            <param name="rate_az" value="1.0"/>
	</node>
        
</launch>

